### PR TITLE
create-calendar

### PR DESCRIPTION
**Pull request by Prokopovych16**
- [DEMO LINK](https://Prokopovych16.github.io/layout_calendar/)

### CODE REVIEW

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Align the calendar to the center of the screen (Not to the top)
image
Fix the weekday selector.
image
Fix day count selector

**Review by etojeDenys — CHANGES_REQUESTED** [review]
you forgot to add "px" here
image


this how your  app must work: [link](https://www.loom.com/share/3dcf1ba5147f45139ef2d1a0d340b0d3?sid=905026f1-aba6-4842-9578-9138943410f9)

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+@each $index, $day in $start-days {
+  .calendar--start-day--#{$day} :first-child {
+    margin-left: 100px * $index + $index;
+  }
+}
+
+@each $index in $month-length {
+  @for $i from $index through $num-days {
+    .calendar--month-length--#{$index} :nth-child(#{$i + 1}) {
+      opacity: 0;
```
Comment (anastasiia-tilikina) [inline src/styles/main.scss]: it is better to use display: none here. I still can hover over hidden days and have cursor pointer

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
GJ!

Let's just fix some tiny issue here)

**Review by ghost — APPROVED** [review]
GJ!

**Pull request closed without merging**